test/system/system.test.ts

```diff
@@ -357,7 +362,12 @@ test("inspect redis:3.2.11-alpine", (t) => {
     .then((imageId) => {
       expectedImageId = imageId;
       return plugin.inspect(img, dockerFileLocation, {
-        manifestGlobs: ["/etc/redhat-release*", "/etc/foo", "/nonexist/bar"],
+        manifestGlobs: [
+          "/etc/redhat-release*",
```

Comment:
We should also test globs which are not for a specific directory. So the DRA uses for example `"**/package.json"` or the exclusion of `**/node_modules/**`. Unfortunately, when it comes to manifest files, we can't assume in which path we will find them.

Reply:
The current list of globs which is used in production can be found here: https://github.com/snyk/docker-registry-agent/blob/develop/src/routes/discover.ts#L160